fish_prompt.fish

```diff
@@ -367,6 +368,23 @@ function __bobthefish_prompt_dir -d 'Display a shortened form of the current dir
   __bobthefish_path_segment "$PWD"
 end
 
+function __bobthefish_prompt_vi -d 'Display vi mode'
+  [ "$theme_display_vi" = 'no' ]; and return
+  set -l vi_mode
+  switch $fish_bind_mode
+  case default
+    __bobthefish_start_segment $__bobthefish_med_grey $__bobthefish_dk_grey --bold
+    echo -n -s 'N'
```

Comment:
People have a very different opinion of what the default mode is though. I always start in insert mode and not in normal mode for example.

vim-airline always shows mode, regardless which mode that is. I added an option to hide a single mode, but that would still mean that the user has to add some configuration if he wants to hide it.